src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -119,28 +119,27 @@ if(WIN32)
     )
 else()
     list(APPEND CORECLR_LIBRARIES
-        ${START_WHOLE_ARCHIVE} # force all PAL objects to be included so all exports are available  
+        ${START_WHOLE_ARCHIVE} # force all PAL objects to be included so all exports are available
         coreclrpal
-        tracepointprovider
-        ${END_WHOLE_ARCHIVE} 
+        ${END_WHOLE_ARCHIVE}
         mscorrc_debug
         palrt
     )
 endif(WIN32)
 
-if(CLR_CMAKE_PLATFORM_UNIX AND FEATURE_EVENT_TRACE)
+if(FEATURE_EVENT_TRACE)
     list(APPEND CORECLR_LIBRARIES
         eventprovider
-  )
-endif(CLR_CMAKE_PLATFORM_UNIX AND FEATURE_EVENT_TRACE)
-
-if(CLR_CMAKE_PLATFORM_LINUX)
-    list(APPEND CORECLR_LIBRARIES
         eventpipe
-  )
-endif(CLR_CMAKE_PLATFORM_LINUX)
-
-target_link_libraries(coreclr ${CORECLR_LIBRARIES}) 
+    )
+    if(CLR_CMAKE_PLATFORM_LINUX)
+        list(APPEND CORECLR_LIBRARIES
+            tracepointprovider
```

Comment:
Nit: Too many spaces before tracepointprovider.

Reply:
It's nested two functions deep, so that seems like the right level of indentation to me

Reply:
Yes, you're right.  Ignore me.  :)